server/seeders/20170602115345-user.js

```diff
@@ -0,0 +1,41 @@
+const bcrypt = require('bcrypt');
+
+module.exports = {
```

Comment:
If you are using this file to set up the initial users (Admin, SuperAdmin) then it would be safer to inject their values through environment variables at runtime, if not, then this should be fine.